### DISCUSSION TITLE
Does EssentialsX 2.18.2 support 1.17.1 ?

### DISCUSSION THREAD

**Discussion by Orholt** [EssentialsX/Essentials#4439] [Q&A] [answered]
Hi does EssentialsX 2.18.2 support Paper 1.17.1 ?

**Comment by Kamil02167** [comment, marked as answer]
> Hi does EssentialsX 2.18.2 support Paper 1.17.1 ?

Yes, no problem.

**Comment by Chew** [comment]
To an extent. It runs fine, but 1.17 items and blocks won't show up with `/i` or `/mob`. There really shouldn't be a reason to not run the dev builds, so why not go ahead and grab them from https://essentialsx.net 😄
